### PR TITLE
Responsive services 1

### PR DESCRIPTION
**Pull request by DanielGurevich1**
responsive fixed
added icons 

icon color remained not solved

### CODE REVIEW

File: css/components/layout.css [hunk 1]
```diff
@@ -80,11 +80,12 @@ body {
             .right.column.col-7 {
                 margin-top: 0;
             }
+        
 }
 
-@media (max-width: 860px) {
+@media (max-width: 840px) {
```
Comment (laurako14) [inline css/components/layout.css]: net nejuokauk taip

File: index.html [hunk 2]
```diff
@@ -345,7 +351,9 @@ <h2 class="col-12 center underline-title services">Our Latest Projects</h2>
     <!-- section NEWS begin -->
     <section class="container">
         <div class="row block-news">
-            <i class="col-4"></i> <!-- ka darysim su sitais, karpysim ar ieskosim ikonu? -->
+            <i class="col-4">
+                <img class="envelope" src="./img/icon/email.png" alt="envelope">
+            </i> <!-- ka darysim su sitais, karpysim ar ieskosim ikonu? -->
```
Comment (laurako14) [inline index.html]: ne pagal uzduoti, mano PR ikeltas su news bus git conflict

**Review by laurako14 — DISMISSED** [review]
Sugriuvus 5 kortele, sugriautas hero responsive, del news bus git conflictas su mano PR, nes darai ne pagal uzduoti

File: index.html [hunk 3]
```diff
@@ -11,6 +11,9 @@
 </head>
 
 <body>
+    <div class="wrapper" id="wrap">
+        <span class="loader"><img class="loader-image" src="./img/preloader/load.png" alt="fun"></span>
+    </div>
```
Comment (andrejusnec) [inline index.html]: O kam cia sita parasei? Man atrodo bus git conflictai kai merginsis kitas branchas, su animacija

**Review by andrejusnec — COMMENTED** [review]
Daniel, o kam originaliam layout tu keiti luzio taskus responsiv?

File: index.html (hunk 3 above)
Comment (DanielGurevich1) [reply index.html]: cia ne mano kodas

**Review by laurako14 — APPROVED** [review]

**Review by andrejusnec — APPROVED** [review]

**Pull request merged**